apps/src/javalab/JavalabView.jsx

```diff
@@ -182,7 +183,12 @@ class JavalabView extends React.Component {
   };
 
   getInstructionsHeight = () => {
-    if (this.props.isVisualizationCollapsed || !this.props.visualization) {
+    if (this.props.isInstructionsCollapsed) {
+      return 30;
```

Comment:
Would it make sense for the instructions view to pass its collapsed height to Javalab (via redux)? Or will this always be 30?

Reply:
You did notice the line I felt most iffy about! :). That said, I'm pretty confident this value will hold steady, so I think I'm okay leaving it hardcoded to get this out before tomorrow's launch build.